src/libraries/System.Net.Http/tests/StressTests/HttpStress/Program.cs

```diff
@@ -177,6 +182,22 @@ private static async Task<ExitCode> Run(Configuration config)
             Console.WriteLine("Query Parameters: " + config.MaxParameters);
             Console.WriteLine();
 
+            if (config.HttpVersion == HttpVersion.Version30 && IsQuicSupported)
+            {
+                Type msQuicApiType = typeof(QuicConnection).Assembly.GetType("System.Net.Quic.MsQuicApi");
```

Comment:
I would prefer to place a comment here about what is this code is doing and why, so it's obvious for someone new to the stress code.